README.rst

```diff
@@ -174,6 +174,21 @@ change:
         return ImageBlockConverter(**options).convert(html)
 
 
+    from markdownify import MarkdownConverter
```

Comment:
Wow, looks great!

Minor nit: these should be two separate blocks; as they are currently, they run in to eachother suggesting they're "one" solution.

Reply:
I agree with you! Should we have 2 examples at all @matthewwithanm? If yes, is there a way to split the code blocks with the indented syntax? I usually use the fenced suntax 😄

Reply:
I haven't tried, but what happens if you just do a newline between the blocks and start the newly-added line with:

```
.. code:: python
```

?